.changeset/pre.json

```
@@ -8,14 +8,11 @@
     "next-example": "0.0.1",
     "vue-example": "0.0.1",
     "amplify-ui-angular-mono": "0.0.1",
+    "@aws-amplify/ui": "0.0.0",
```

Let's set this at 3.0.0, so that it's easier for changeset to infer the next version once we exit pre-release mode. I did this for `@aws-amplify/ui-angular` below.

---

Then we can set https://github.com/aws-amplify/amplify-ui/blob/51b37e27a38d6a8c936993322186d7895b36e685/packages/ui/package.json#L4 to 3.0.1-next.0. When we run `changeset pre exit`, it'll be at 3.0.1!

---

If this looks good, commit away 🙏 

```suggestion
    "@aws-amplify/ui": "3.0.0",
```

---

Let's do it! And I'll do the other one in `ui/package.json`.